charts/azuremonitor-containers/values.yaml

```diff
@@ -69,6 +69,12 @@ amalogs:
   # This flag to enable and disable custom metrics and default is enabled
   enableCustomMetrics: true
 
+  # This flag to enable and disable containerlogv2 KubernetesMetadataCollection and default is enabled
```

Comment:
We dont require these since these are already configurable through config map.

Reply:
changed